cdap-ui/app/features/hydrator/controllers/create/toppanel-ctrl.js

```diff
@@ -87,38 +87,25 @@ class TopPanelController{
       templateUrl: '/assets/features/hydrator/templates/create/popovers/viewconfig.html',
       size: 'lg',
       keyboard: true,
-      controller: ['$scope', 'config', function($scope, config) {
+      controller: ['$scope', 'config', '$timeout', 'exportConfig', function($scope, config, $timeout, exportConfig) {
         $scope.config = JSON.stringify(config);
-
+        var exportConf = exportConfig;
         $scope.export = function () {
-          // CanvasFactory
-          //   .exportPipeline(
-          //     MyAppDAGService.getConfigForBackend(),
-          //     MyAppDAGService.metadata.name,
-          //     MyAppDAGService.nodes,
-          //     MyAppDAGService.connections)
-          //   .then(
-          //     function success(result) {
-          //       $scope.exportFileName = result.name;
-          //       $scope.url = result.url;
-          //       $scope.$on('$destroy', function () {
-          //         URL.revokeObjectURL($scope.url);
-          //       });
-          //       // Clicking on the hidden download button. #hack.
-          //       $timeout(function() {
-          //         document.getElementById('pipeline-export-config-link').click();
-          //       });
-          //     }.bind(this),
-          //     function error() {
-          //       console.log('ERROR: Exporting ' + MyAppDAGService.metadata.name + ' failed.');
-          //     }
-          //   );
+          var content = exportConf;
```

Comment:
this is kind of replicating exportConfig right? Do we need to assign it to a different variable?